cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -2695,6 +2701,28 @@
 (decl x64_pcmpgtq (Xmm XmmMem) Xmm)
 (rule (x64_pcmpgtq x y) (xmm_rm_r $I64X2 (SseOpcode.Pcmpgtq) x y))
 
+;; Helpers for read-modify-write ALU form (AluRM).
+(decl alu_rm (Type AluRmiROpcode Amode Gpr) SideEffectNoResult)
+(rule (alu_rm ty opcode src1_dst src2)
+      (let ((size OperandSize (operand_size_of_type_32_64 ty)))
+        (SideEffectNoResult.Inst (MInst.AluRM size opcode src1_dst src2))))
+
+(decl x64_add_mem (Type Amode Gpr) SideEffectNoResult)
+(rule (x64_add_mem ty addr val)
+      (alu_rm ty (AluRmiROpcode.Add) addr val))
+(decl x64_sub_mem (Type Amode Gpr) SideEffectNoResult)
```

Comment:
Newlines between these for readability and consistency with others

Reply:
Fixed.